### ISSUE TITLE
Bug Report: Not able to create advertisement.

### ISSUE THREAD

**Issue by skbhagat0502** [PalisadoesFoundation/talawa-admin#1627]
**Describe the bug**
When we try to create a new advertisement, we get an error. 
**To Reproduce**

1.Login to the talawa admin.
2.Go to the advertisement page.
3.Try to create a new advertisement.
4. You will get an error.

**Expected behavior**
We should be able to create advertisement without any error.

**Actual behavior**
A clear and concise description of how the code performed w.r.t expectations.

**Screenshots**

https://github.com/PalisadoesFoundation/talawa-admin/assets/109683163/d7f24601-a2a5-4ed1-b797-b8d2895ffc61

**Additional details**

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

**Comment by skbhagat0502** [comment]
@palisadoes Please assign me this issue.

**Comment by Cioppolo14** [comment]
@skbhagat0502  Are you working in the develop branch and still seeing this? Right now we're waiting on a fix to be able to use Talawa-admin which should be done today, but I had not seen the advertisement issue in the past.

**Comment by disha1202** [comment]
@skbhagat0502 Can you please make sure that Talawa-API develop branch is updated with the latest upstream on you local and test it again.

**Comment by skbhagat0502** [comment]
@disha1202 After updating my branch I am not able to navigate to the org dashboard.

https://github.com/PalisadoesFoundation/talawa-admin/assets/109683163/288bd84c-fec7-4f86-9fd9-5c8de8446a23

**Comment by skbhagat0502** [comment]
@Cioppolo14 Please have a look into this.

**Comment by skbhagat0502** [comment]
@Cioppolo14 After the sync I found that this issue is not resolved but happening due to some other problem. Closing this issue as some other problem is causing this. Creating a new issue for that.
